config/initializers/sentry.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# This file was previously config/initializers/raven.rb
+
+require 'sentry/event_scrubber'
+
+# This custom Transport class is need to log the error
+# This needs to be manually tested
+# https://github.com/getsentry/sentry-ruby/issues/1583
+transport = Class.new(Sentry::HTTPTransport) do
+  def send_data(data)
+    super
+  rescue Sentry::ExternalError => e
+    Rails.logger.error(
+      "Error performing Sentry#send_data: #{e.message}",
+      original_event: data
+    )
+    StatsD.increment('worker.sentry.error')
+  end
+end
+
+Rails.application.reloader.to_prepare do
+  Sentry.init do |config|
+    config.dsn = Settings.sentry.dsn if Settings.sentry.dsn
+    config.environment = Settings.vsp_environment if Settings.vsp_environment
+    # No longer need to ignore 'ActionController::InvalidAuthenticityToken'
+    # https://github.com/getsentry/sentry-ruby/blob/master/sentry-ruby/lib/sentry/configuration.rb#L354
+
+    # config.async is deprecated and could get removed
+    # https://github.com/getsentry/sentry-ruby/issues/1522
```

Comment:
Could this comment be removed? Are we using `config.async` anywhere in the codebase?